FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -162,22 +170,27 @@ else if (settingList.isEmpty())
                             }
                             else
                             {
-                                configInts[8] = -1;
+                                configInts[8] = 0;
                             }

```

Comment:
Last version is fully compatible, but -1 isn't a legal damage in minecraft. If the old configuration file contains a damage -1, it will be set to 0. With -1 it is not possible to find a block in ore dictionary.
The new searching is:
- Search all IDs in ore dictionary for given block. If a damage is set it will be used. Otherwise it searches all IDs for different damages of same block.
- Create CompassTargetData for all located blocks.
- It also works for block, these are not registered in ore dictionary.

New function during searching:
- Use flag useDamage instead check damage against -1 in ThreadCompassWorker